vendor.conf

```diff
@@ -1,57 +1,113 @@
-github.com/Azure/go-ansiterm 388960b655244e76e24c75f48631564eaefade62
-github.com/Microsoft/go-winio v0.4.2
-github.com/Nvveen/Gotty a8b993ba6abdb0e0c12b0125c603323a71c7790c https://github.com/ijc25/Gotty
-github.com/Sirupsen/logrus v0.11.0
-github.com/agl/ed25519 d2b94fd789ea21d12fac1a4443dd3a3f79cda72c
-github.com/coreos/etcd 824277cb3a577a0e8c829ca9ec557b973fe06d20
-github.com/cpuguy83/go-md2man a65d4d2de4d5f7c74868dfa9b202a3c8be315aaa
-github.com/davecgh/go-spew 346938d642f2ec3594ed81d874461961cd0faa76
-github.com/docker/distribution b38e5838b7b2f2ad48e06ec4b500011976080621
-github.com/docker/docker d58ffa0364c04d03a8f25704d7f0489ee6cd9634
-github.com/docker/docker-credential-helpers v0.5.1
+# docker
```

Comment:
I think we should leave this file sorted. It makes it easier to maintain and avoid conflicts when multiple PRs change this file.  Commented sections are difficult to maintain and will lead to an unsorted mess.  I guess there were a few lines that were not sorted previously. I'll see about adding a validation for this file to keep it sorted.

I checked out this PR and re-sorted the file to look at the vendor changes. I noticed that we have a bunch of new dependencies that I don't think we actually use. Somewhere in the dependency there are some bad packages that are overloaded and forcing us to pull in a bunch of unnecessary extra libraries.  

I'm going to look into this to see if we can remove some of these.